pkg/filters/util.go

```diff
@@ -0,0 +1,52 @@
+package filters
+
+import (
+	"github.com/flyteorg/flytectl/cmd/config"
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
+)
+
+func BuildResourceListRequestWithName(c Filters, name string) (*admin.ResourceListRequest, error) {
+	fieldSelector, err := Transform(SplitTerms(c.FieldSelector))
+	if err != nil {
+		return nil, err
+	}
+	request := &admin.ResourceListRequest{
+		Limit:   uint32(c.Limit),
+		Filters: fieldSelector,
+		Id: &admin.NamedEntityIdentifier{
+			Project: config.GetConfig().Project,
+			Domain:  config.GetConfig().Domain,
+		},
```

Comment:
have this information passed to this function instead of fetched from the config